packages/cli/package.json

```
@@ -34,9 +34,9 @@
     "dist/*"
   ],
   "scripts": {
-    "build": "tsdown src/index.ts",
-    "dev": "TSUP_DEV=true tsdown --watch src/ src/index.ts",
-    "start": "node dist/index.js",
+    "build": "tsdown",
+    "dev": "TSUP_DEV=true tsdown --watch src/",
+    "start": "node dist/src/index.js",
```

I was checking the current build process and this change is not required. When building it in `development`, it still keeps the `dist/index.js` route (as it should be):

```plain
❯ ls -lah
Permissions Size User  Date Modified Name
.rwxr-xr-x   265 angel 24 sep 12:17  index.js
.rw-r--r--   459 angel 24 sep 12:17  index.js.map
.rw-r--r--  477k angel 24 sep 12:17  program-BMYv-yVz.js
.rw-r--r--  921k angel 24 sep 12:17  program-BMYv-yVz.js.map
drwxr-xr-x     - angel 24 sep 12:17  utils
```

I mentioned it should be this way to avoid differences with the "production" builds. In production, the `dist` folder only contains the `index.js` file.

You can keep this command as it is:

```suggestion
    "start": "node dist/index.js",
```